### ISSUE TITLE
Feature/Bug: TRPC generator should support "Unchecked" Inputs

### ISSUE THREAD

**Issue by bvkimball** [zenstackhq/zenstack#499]
Been looking at the TRPC generated api.  Noticed the api uses the create inputs... but generates the UncheckedInputs as well... are they not used. Can they be? Should they be?

ie..
```diff
  create: z.object({
    select: z.lazy(() => CategorySelectObjectSchema.optional()),
    include: z.lazy(() => CategoryIncludeObjectSchema.optional()),
-    data: CategoryCreateInputObjectSchema,
+    data: z.union([CategoryCreateInputObjectSchema, CategoryUncheckedCreateInputObjectSchema]),
  }),
```

Obviously needs to be supported in the all methods: create, update, upsert etc.

**Comment by ymc9** [comment]
Thanks for filing this @bvkimball . Just wondering about your use case of unsafe input types. My understanding is that it allows you to pass in scalar id for relation fields during creation, but I'm not sure if there's actually more about it.

**Comment by ymc9** [comment]
Fixed by #512
